uint8 speed        # Speed (0-255, 0 to use current setting)
---
bool success       # True if command was successful